.packit.yaml

```
@@ -4,21 +4,45 @@ upstream_package_name: fapolicyd
 downstream_package_name: fapolicyd
 upstream_tag_template: v{version}
 
-actions:
-    post-upstream-clone:
-      - bash -c "sed -i 's/#ELN //' fapolicyd.spec"
-      - bash -c "curl -L -o $(grep Source1 fapolicyd.spec | cut -d ' ' -f 2 | sed -r \"s/%\{name\}/$(grep 'Name:' fapolicyd.spec | cut -f 2 -d ' ')/g;s/%\{semodule_version\}/$(grep '%define semodule_version' fapolicyd.spec | cut -f 3 -d ' ')/g\" | sed -r 's|(.*)#/(.*)|\2 \1|')"
-      - bash -c "curl -L -o $(grep Source2 fapolicyd.spec | cut -d ' ' -f 2 | sed -r 's|(.*)#/(.*)|\2 \1|')"
-
 jobs:
   - job: copr_build
     trigger: pull_request
+    identifier: build-normal
+    actions:
+      post-upstream-clone:
+        - bash -c "sed -i 's/#ELN %//' fapolicyd.spec"
```

This might be potentially causing the missing tarball issue.
```suggestion
        - bash -c "sed -i 's/#ELN //' fapolicyd.spec"
```